mne/preprocessing/tests/test_stim.py

```diff
@@ -51,9 +51,6 @@ def test_fix_stim_artifact():
 
     # use window before stimulus in raw
     event_idx = np.where(events[:, 2] == 1)[0][0]
-    tmin, tmax = -0.045, -0.015
-    tmin_samp = int(-0.035 * raw.info['sfreq'])
-    tmax_samp = int(-0.015 * raw.info['sfreq'])
```

Comment:
they are already defined. I don't know why they were redefined here -- it seemed redundant to me.